lib/lte_link_control/lte_lc.c

```diff
@@ -630,6 +957,262 @@ static int parse_nw_reg_status(const char *at_response,
 	return err;
 }
 
+/**@brief Parses an AT command response, and returns the current RRC mode.
+ *
+ * @param at_response Pointer to buffer with AT response.
+ * @param mode Pointer to where the RRC mode is stored.
+ * @param mode_index Parameter index for mode.
+ *
+ * @return Zero on success or (negative) error code otherwise.
+ */
+static int parse_rrc_mode(const char *at_response,
+			  enum lte_lc_rrc_mode *mode,
+			  size_t mode_index)
+{
+	int err, temp_mode;
+	struct at_param_list resp_list = {0};
+
+	if ((at_response == NULL) || (mode == NULL)) {
+		return -EINVAL;
+	}
+
+	err = at_params_list_init(&resp_list, AT_CSCON_PARAMS_COUNT_MAX);
+	if (err) {
+		LOG_ERR("Could not init AT params list, error: %d", err);
+		return err;
+	}
+
+	/* Parse CSCON response and populate AT parameter list */
+	err = at_parser_max_params_from_str(at_response,
+					    NULL,
+					    &resp_list,
+					    AT_CSCON_PARAMS_COUNT_MAX);
```

Comment:
Same comment as previous: last parameter `AT_CSCON_PARAMS_COUNT_MAX` is not needed.